wandb/sdk/verify/verify.py

```diff
@@ -61,7 +61,6 @@ def check_host(host: str) -> bool:
 
 
 def check_logged_in(api: Api, host: str) -> bool:
-    print("Checking if logged in".ljust(72, "."), end="")
```

Comment:
Probably can revert all changes to this file

Reply:
Any `git` tips for resetting specific files from previous commits?

Reply:
`git checkout master -- wandb/sdk/verify/verify.py`

Reply:
more generally:
`git checkout <branch> -- file/from/branch.py`